### PR TITLE
Page Header Refactor

### PR DESCRIPTION
**Pull request by ashenden**
Dependent on this PR (https://github.com/dcos/dcos-ui/pull/649)

This work centers around changes to the Page Header `.page-header`.

* Hiding the sidebar toggle CTA (hamburger menu) in favor of a yet to be built future concept
* Introduced `page-header-navigation` which will replace the previous navigation tabs
* New `menu-tabbed` component supports simple tabs that work almost everywhere are now used in `page-header-navigation`.
* Removed border from sidebar
* Upgrade CNVS to support patches
* Minor tweaks to the `.header`

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

**Comment by ashenden** [comment]
Merged in the updates made in the `cnvs-upgrade/initial-layout` branch.  Fire away!

**Comment by MatApple** [comment]
Ouch!

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "browser-info": "0.4.0",
-    "cnvs": "1.0.0",
+    "cnvs": "1.0.2",
```
Comment (kennyt) [inline package.json]: this needs to be shrinkwrapped too. @jfurrow can help

**Pull request closed without merging**